dev/openchpl_data-model.sql

```diff
@@ -27,6 +27,24 @@ CREATE TYPE openchpl.attestation as enum('Affirmative', 'Negative', 'N/A');
 CREATE TYPE openchpl.validation_message_type as enum('Error', 'Warning');
 CREATE TYPE openchpl.job_status_type as enum('In Progress', 'Complete', 'Error');
 
+CREATE TABLE openchpl.chart_data(
+	chart_data_id bigserial NOT NULL,
```

Comment:
A lot of the original tables use these long id column names but in all the newer ones I've been creating for a while I just call it "id". I'm not sure if we'll ever get to go back and change the old ones but could you make these match the newer style? The reason is that PSQL automatically creates a sequence for bigserial types using the pattern table_column_seq, so all the sequences with this longer style id have names like "chart_data_chart_data_id_seq" which is awkward. If we just call the column "id" then the generated sequence is named "chart_data_id_seq" which is much nicer to read.